src/modules/companies-house/routes.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Joi = require('@hapi/joi');
+const controller = require('./controller');
+
+module.exports = {
+  getCompaniesHouseCompanies: {
+    path: '/water/1.0/companies-house/search/companies',
+    method: 'GET',
+    handler: controller.getCompaniesHouseCompanies,
+    config: {
+      description: 'Gets a list of returns for the specified company',
```

Comment:
Is this description correct? Should it be something like: 'Gets a list of companies for the specified name'